src/api/l_audio.c

```diff
@@ -150,15 +157,43 @@ static const luaL_Reg lovrAudio[] = {
   { "setCaptureDevice", l_lovrAudioSetCaptureDevice },
   { "getCaptureDevice", l_lovrAudioGetCaptureDevice },
   { "listCaptureDevices", l_lovrAudioListCaptureDevices },
+  { "getSpatializerName", l_lovrAudioGetSpatializerName },
   { NULL, NULL }
 };
 
 int luaopen_lovr_audio(lua_State* L) {
   lua_newtable(L);
   luax_register(L, lovrAudio);
   luax_registertype(L, Source);
-  AudioConfig config[2] = { { .enable = true, .start = true }, { .enable = false, .start = false } };
-  if (lovrAudioInit(config)) {
+
+  char *spatializer = NULL;
+  int spatializerMaxSourcesHint = AUDIO_SPATIALIZER_MAX_SOURCES_HINT;
+  luax_pushconf(L);
+  lua_getfield(L, -1, "audio");
+  if (lua_istable(L, -1)) {
+    lua_getfield(L, -1, "spatializerMaxSourcesHint");
+    if (lua_type(L, -1) == LUA_TNUMBER)
+      spatializerMaxSourcesHint = lua_tointeger(L, -1);
+    lua_pop(L, 1);
```

Comment:
If LÖVR was like "the max number of sources is always 64" would this help simplify some of this stuff surrounding the max source count

Reply:
If LOVR set that requirement then we could remove almost all this logic from oculus spatializer (effectively because the source-limit logic would then be in audio.c) but there would still be a little bit of ugliness because oculus spatializer *does* have "tails", audio.c doesn't know about the tails, and the tails are bound to ID numbers. I don't think this is worth removing capability from apps not using this spatializer.